docs/user/extra-js.rst

```diff
@@ -0,0 +1,42 @@
+Loading extra javascript files
```

Comment:
Same here regarding capitalization, and any other heading.

You can avoid to capitalize conjunctions (and, with), prepositions (from, for) articles (a, the).